SU-SOE/stanford_soe_helper
# 7.x 2.x spotlight dropdowns css

# NOT READY FOR REVIEW

# Summary
Styling for the Spotlight filters

# Needed By (Date)
Holding until we're clear on what should happen with Spotlight

# Criticality
- Fixes broken stuff


# Steps to Test
- Switch to this branch
- Navigate to /spotlight
- Verify you see what is documented and decided upon in [SOE-2939](https://stanfordits.atlassian.net/browse/SOE-2939)

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2939

## Related PRs
This is branched from https://github.com/SU-SOE/stanford_soe_helper/pull/170

## More Information

## Folks to notify



# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
